my_project/tab_outdoor_comfort/app_outdoor_comfort.py

```diff
@@ -118,6 +117,7 @@ def layout_outdoor_comfort():
                     )
                 ],
             ),
+            html.Div(id='output'),
```

Comment:
Please use a more meaningful name for the output. Since this is not very informative and someone else may use it in other sections of the application. Have a look at the other ids. We were using the following convention `id="month-hour-filter-outdoor-comfort" where the first part describes the functionality and the second part of the id is stating the name of the tab in which it appears.